test-support/README.md

```diff
@@ -0,0 +1,5 @@
+# OIDC
+
+Provides an OIDC implementation used in testing based on [node-oidc-provider](https://github.com/panva/node-oidc-provider).
+
+Hard codes various client configuration values that users will need to also set.
```

Comment:
suggestion (non-blocking): recommend adding the docker command that builds the image